src/ngAnimate/module.js

```diff
@@ -267,9 +267,22 @@
  * .message.ng-enter-prepare {
  *   opacity: 0;
  * }
- *
  * ```
  *
+ * ### Animating between subsequent value changes
```

Comment:
Better thatn before, but I think "arbitrary" instead of "subsequent" would be preferable (because you always want to animate "subsequent changes").

Reply:
Yeah, I had [this meaning][1] in mind:

> Mathematics. undetermined; not assigned a specific value:
> an arbitrary constant.

Maybe "unknown"?

[1]: http://www.dictionary.com/browse/arbitrary

Reply:
Not sure about this. Maybe it should be just `Animating between value changes`?